contrib/python-acme/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "python-acme"
+pkgver = "2.9.0"
+pkgrel = 0
+build_wrksrc = "acme"
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-wheel",
+]
+depends = [
+    "python-cryptography",
+    "python-josepy",
+    "python-openssl",
+    "python-pytz",
+    "python-requests",
+    "python-pyrfc3339",
+]
+checkdepends = [
+    "python-pytest",
+] + depends
+pkgdesc = "ACME protocol implementation"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "Apache-2.0"
+url = "https://github.com/certbot/acme"
```

Comment:
this url is a 404

Reply:
ah, probably should be https://github.com/certbot/certbot/tree/master/acme ?
As its inside the certbot project

Reply:
hmmm, sure, can't think of anything better